package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-kms-http",
   "dependencies": {
+    "bedrock-validation": "^4.4.0",
```

Comment:
bedrock modules are `peerDependencies` for other bedrock modules.  In the test suite however, those same peerDeps will be regular deps.